sdk/src/main/java/com/horizen/account/api/rpc/types/TransactionArgs.java

```diff
@@ -0,0 +1,160 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.horizen.account.api.rpc.handler.RpcException;
+import com.horizen.account.api.rpc.utils.RpcCode;
+import com.horizen.account.api.rpc.utils.RpcError;
+import com.horizen.account.proposition.AddressProposition;
+import com.horizen.account.state.Message;
+import com.horizen.account.transaction.EthereumTransaction;
+import com.horizen.account.utils.BigIntegerUtil;
+import com.horizen.evm.utils.Address;
+import org.web3j.crypto.Sign;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TransactionArgs {
+    public BigInteger type;
+    public Address from;
+    public Address to;
```

Comment:
Is it ok to use `Address` type here?
or `byte[]` or String or `AddressProposition.
To be discussed.